db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Operations/StateMachines/ResizeRequest.class/__methods__/pending_request.rb

```diff
@@ -0,0 +1,11 @@
+# Second step - wait until instance get ready for resize confirm
+
+vm = $evm.root['vm']
+
+if vm.validate_resize_confirm
```

Comment:
Initial resize call changes state of VM, once state is updated in MIQ database, repeated call is not possible because of validation. If the validation is skipped, resize call on already resizing VM should fail immediately on openstack nova side. So, there cannot be two resize processes in progress in same time on one VM AFAIK.